string filename
bool lossy
---
int64 size
bool result
